website/docs/core/certificates.md

```diff
@@ -69,6 +70,24 @@ ak import_certificate --certificate /certs/mycert.pem --private-key /certs/somet
 
 This will import the certificate into authentik under the given name. This command is safe to run as a cron job; authentik will only re-import the certificate if it changes.
 
+## Configure a webhook to use uploaded SSL certificates
+
+When communicating with an external API, you will need to configure a generic webhook to utilize uploaded SSL certificates (CA).
+
+1. Create a custom image with a Docker file that looks something like this:
+
```

Comment:
I know we go for a "friendly, communal" tone, but this feels loose.  Suggestion: "Create a custom image with a Docker file.  For example:"